options.go

```diff
@@ -161,16 +164,39 @@ func (o *Options) Validate() error {
 	}
 
 	if o.OIDCIssuerURL != "" {
-		// Configure discoverable provider data.
-		provider, err := oidc.NewProvider(context.Background(), o.OIDCIssuerURL)
-		if err != nil {
-			return err
+
+		ctx := context.Background()
+
+		// Override discoverable provider data with a custom http.Client that fakes
```

Comment:
This comment is now out of date, could you update it please

Reply:
done :)